operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,110 @@
+# Audit Logs
+
+## Enable or disable audit logging for host and Kubernetes APIs
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional. If enabled it generates some load and data on the non-compute nodes \(NCNs\).
+By default, host and Kubernetes API audit logging are not enabled.
+
+### Procedure
+
+The method for updating the audit log settings varies depending on the state of the system.
+
+1. Select one of the following options based on the installation status of the system:
+
+   * Install
+
+     To update the audit log settings during the installation, use one of the following options:
+
+     * **Option 1**
+
+       During the installation, audit logging is enabled or disabled by modifying the CSI settings.
+       To enable or disable audit logging, use the following flags with the `csi config init` command. For more information on using flags, see `csi config init -h`:
+
+       `ncn-mgmt-node-auditing-enabled`: Set to `true` to enable host logging or to `false` to disable host logging.
+
+       ```console
+       ncn# csi config init --ncn-mgmt-node-auditing-enabled=true [other config init options]
+       ```
+
+       `k8s-api-auditing-enabled`: Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.
+
+       ```console
+       ncn# csi config init --k8s-api-auditing-enabled=true [other config init options]
+       ```
+
+     * **Option 2**
+
+       Adjust the audit log settings by editing the `system_config.yaml` file.
+
+       View the current settings with the following command:
+
+       ```bash
+       ncn# cd /var/www/ephemeral/prep
+       ncn# grep audit system_config.yaml
+       ```
+
+       Example output:
+
+       ```text
+       k8s-api-auditing-enabled: false
+       ncn-mgmt-node-auditing-enabled: false
+       ```
+
+   * Post-install
+
+     If a system is installed, audit logging can be enabled through the `csi` command:
+
+     * `ncn-mgmt-node-auditing-enabled`
```

Comment:
Are users supposed to perform both this bulleted step and the next one? If these are multiple steps and the user is supposed to do both of them, I recommend changing these to numeric lists, so that is clearer.